docs/LDAP.rst

```diff
@@ -52,7 +52,7 @@ framework <https://www.jboss.org/picketlink/IDM>`__ that allows a very
 flexible integration with an LDAP directory:
```

Comment:
We should rename this chapter IMO, this can let think that there is another way to integrate with LDAP/AD.
Actually I think that all sub-chapter (Quick start, Configuration review, ...) must be directly put under the main chapter "LDAP Integration" and the content of "LDAP integration using PicketLink" should be moved in a sibling chapter simply called "PicketLink".
WDYT ?

Reply:
OK